From segooon@gmail.com  Thu Jul 22 15:52:47 2010
From: Kulikov Vasiliy <segooon@gmail.com>
To: kernel-janitors@vger.kernel.org
Cc: Greg Kroah-Hartman <gregkh@suse.de>,
	Arnd Bergmann <arnd@arndb.de>,
	Jesper Nilsson <jesper.nilsson@axis.com>,
	Ingo Molnar <mingo@elte.hu>, linux-kernel@vger.kernel.org
Subject: serial: crisv10: formatting of pointers in printk()
Date: Wed, 14 Jul 2010 22:04:42 +0400
Message-Id: <1279130682-13704-1-git-send-email-segooon@gmail.com>

Use %p instead of %08x in printk().

Signed-off-by: Kulikov Vasiliy <segooon@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/serial/crisv10.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/serial/crisv10.c
+++ b/drivers/serial/crisv10.c
@@ -4522,8 +4522,8 @@ static int __init rs_init(void)
 		INIT_WORK(&info->work, do_softint);
 
 		if (info->enabled) {
-			printk(KERN_INFO "%s%d at 0x%x is a builtin UART with DMA\n",
-			       serial_driver->name, info->line, (unsigned int)info->ioport);
+			printk(KERN_INFO "%s%d at %p is a builtin UART with DMA\n",
+			       serial_driver->name, info->line, info->ioport);
 		}
 	}
 #ifdef CONFIG_ETRAX_FAST_TIMER
